mlflow/tensorflow/callback.py

```diff
@@ -0,0 +1,99 @@
+from tensorflow import keras
+
+from mlflow import log_params, log_text
+from mlflow.utils.autologging_utils import BatchMetricsLogger
+
+
+class MLflowCallback(keras.callbacks.Callback):
```

Comment:
https://output.circle-artifacts.com/output/job/d2c3abe1-cd1f-4244-8f9d-efc457b292d7/artifacts/0/docs/build/html/python_api/mlflow.tensorflow.html

`MLflowCallback` is not documented.

Reply:
It looks like sphinx only documents classes/functions defined in `mlflow/tensorflow/__init__.py`.

Reply:
that is odd, there shouldn't be diff between directly defining inside `__init__.py` and importing inside `__init__.py`. Is there any fix we can do aside from moving it into `__init__.py`?

Reply:
We'll have to get this imported in one way or another as this doesn't show up anywhere in docs references. Can we also add a note to the tensorflow autologger docstring that warns about the behavior of using autologging + the Callback together (as a `warning` type rather than a `note` type) explaining why these can't be run together (duplicated entries for each step)?

Reply:
sg! added info to the docstring.